packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -120,6 +120,7 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     const torrent = this.torrents.find(t => t.infoHash === infoHash);
     if (torrent) {
       torrent.pause(); // the paymentChannelClosing is done at the moment of payment
+      this.closeChannels(torrent, true);
```

Comment:
Just to make sure I understand the current application correctly. This `pause` method that includes this line is never called, right?

Reply:
The `web3torrent.pause()` method is not being used currently, no.